src/Symfony/Component/Security/Guard/AbstractGuardAuthenticator.php

```diff
@@ -19,9 +20,21 @@
  *
  * @author Ryan Weaver <ryan@knpuniversity.com>
  */
-abstract class AbstractGuardAuthenticator implements GuardAuthenticatorInterface
+abstract class AbstractGuardAuthenticator implements AuthenticatorInterface
 {
     /**
+     * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
```

Comment:
If I remove this line, the false positive deprecation notice is removed. Should I open a PR?

Reply:
Please do. This method must be implemented by concrete subclasses and it is unlikely to be called directly.